### ISSUE TITLE
Test against compliance test suite

### ISSUE THREAD

**Issue by glyn** [taimoorzaeem/aeson-jsonpath#1]
Please see this list of resources for RFC 9535 implementers: https://underlap.org/implementing-jsonpath-rfc-9535

**Comment by taimoorzaeem** [comment]
Thanks @glyn for these resources. The complete compliance testing would be done once the implementation is complete.

**Comment by glyn** [comment]
Fair enough, but sometimes tests throw up the need for changes and it's easier to rework a smaller codebase. Each to his/her own though... :-)

Anyway, I really appreciate you taking RFC 9535 seriously.

**Comment by taimoorzaeem** [comment]
> Fair enough, but sometimes tests throw up the need for changes and it's easier to rework a smaller codebase.

Agreed.

**Comment by taimoorzaeem** [comment]
Completed in #22.
